cobc/config.def

```diff
@@ -383,6 +383,12 @@ CB_CONFIG_SUPPORT (cb_record_delim_with_fixed_recs, "record-delim-with-fixed-rec
 CB_CONFIG_SUPPORT (cb_missing_statement, "missing-statement",
 	_("missing statement (e.g. empty IF / PERFORM)"))
 
+CB_CONFIG_SUPPORT (cb_missing_period, "missing-period",
+	_("missing period in PROCEDURE DIVISION (when reference format supports Area A enforcement)"))
+
+CB_CONFIG_SUPPORT (cb_non_word_in_area_a, "non-word-in-area-a",
```

Comment:
What is a "non user-defined word"? Is this is about "statements" or "reserved words" or ???